### ISSUE TITLE
 question about code

### ISSUE THREAD

**Issue by lclszsdnr** [gicheonkang/DAN-VisDial#15]
Hello author, although your article has been published for many years, I still think it is very important, so I read your code recently.
I have one doubts when I read the code, and I want to ask you for advice. At the end of the decoder, you used the logsoftmax function to calculate the probability of the candidate answer, and then input it into the nn.CrossEntropy function, but as far as I know, nn.CrossEntropy function comes with a logsoftmax calculation. What is the purpose of doing this?

**Comment by gicheonkang** [comment]
Hello, sorry for the late response.
Yes, this is our mistake. The log softmax should be deleted, or we must replace CrossEntropy loss with NLLLoss.

This work was conducted in 2018. So, I highly recommend you check our recent visual dialog work (https://github.com/gicheonkang/gst-visdial). Thank you!
